IronDome/wwwroot/css/site.css

```diff
@@ -19,4 +19,8 @@ html {
 
 body {
   margin-bottom: 60px;
+}
+
+.yosi{
```

Comment:
try avoidind your name as a class-name

Reply:
my name is holy bablical name so i can put it wherever i want